cowprotocol/cowswap
# Add new Privacy policy

# Summary

This PR updates Privacy Policy page with a new version defined here https://docs.google.com/document/d/1m9BgMYKDEJ5nIbLAwGcGWxx-S8jBjG_F8MYKO8BGW3A/edit#

# To test
- go to `/privacy-policy` page
- make sure the content is correct as defined in the doc
